### PR TITLE
Fix custom user text playlist does not work

### PR DESCRIPTION
**Pull request by kostyanf14**

### CODE REVIEW

**Review by akihikodaki — COMMENTED** [review]
What if `select_test_names` is not specified?

**Comment by kostyanf14** [comment]
> What if `select_test_names` is not specified?

we have `return unless select_test_names` in `def custom_select_test_names`

**Comment by akihikodaki** [comment]
> > What if `select_test_names` is not specified?
> 
> we have `return unless select_test_names` in `def custom_select_test_names`

Won't we need to set `run_count` in that case too?

**Comment by kostyanf14** [comment]
> Won't we need to set `run_count` in that case too?

We set `run_count` in list_tests for default value 1

File: lib/engines/hcktest/playlist.rb [hunk 1]
```diff
@@ -90,14 +90,16 @@ def sort_by_duration
...
 
       @tests.select! do |test|
         next unless select_test_names_counted.key?(test['name'])
 
-        test['run_count'] = select_test_names_counted[test['name']]
+        test['run_count'] = if @project.options.test.allow_test_duplication
+                              select_test_names_counted[test['name']]
+                            else
+                              1
+                            end
```
Comment (akihikodaki) [inline lib/engines/hcktest/playlist.rb:102]:
```suggestion
        if @project.options.test.allow_test_duplication
          test['run_count'] = select_test_names_counted[test['name']]
        end
        
        true
```

**Review by akihikodaki — CHANGES_REQUESTED** [review]

**Review by akihikodaki — APPROVED** [review]

**Pull request merged**